eng/Versions.props

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MajorVersion>3</MajorVersion>
+    <MinorVersion>0</MinorVersion>
+    <PatchVersion>0</PatchVersion>
+    <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/aspnet/AspNetCore/2.0.0/LICENSE.txt</PackageLicenseUrl>
+    <Serviceable Condition="'$(Configuration)' == 'Release'">true</Serviceable>
+    <UsingToolNetFrameworkReferenceAssemblies Condition="'$(OS)' != 'Windows_NT'">true</UsingToolNetFrameworkReferenceAssemblies>
```

Comment:
@natemcmaster I'm not sure making this conditional on non-windows is the right move, but it's the only way I could figure out to not have binding re-direct issues.

Reply:
I'd rather have this always set to `true`, regardless of host OS. It means we don't have to force all contributors to acquiring a new targeting packs when we update netfx in our tests.

Reply:
It's odd that this would change binding redirects. You need binding redirects regardless of where reference assemblies come from. Setting this to true means the ref assemblies are found in a nuget package, instead of Program Files.

Reply:
@tmat this is the `Cannot find reference assembly 'System.ComponentModel.Composition.dll' file for package System.ComponentModel.Composition` thing that I brought up over email. I'm open to the above being the wrong solution but without this I get the missing reference assembly stuff, so I need an alternative solution or a suggestion on how to produce one.

Reply:
After much debate and concern, we eventually landed on essentially the fix I have above for this kind of problem.